go.mod

```diff
@@ -3,28 +3,28 @@ module github.com/athenianco/cloud-common
 go 1.20
 
 require (
-	cloud.google.com/go/kms v1.8.0
-	cloud.google.com/go/pubsub v1.28.0
-	cloud.google.com/go/storage v1.29.0
-	github.com/getsentry/sentry-go v0.18.0
+	cloud.google.com/go/kms v1.10.1
```

Comment:
Can you please rollback dependency updates? Please try to don't mix these with regular PRs.

Reply:
https://github.com/athenianco/cloud-common/pull/140
new PR with cherry-picked bumps